packages/brand/src/index.css

```diff
@@ -1,7 +1,7 @@
 :where(.cui-brand-icon) {
 	--scale: 1;
 	--aspect-ratio: 1;
-	--square-to-circle-ratio: 1.1284;
+	--square-to-circle-ratio: 1.515;
```

Comment:
It is good enough and there is not much *math* around it—just looks better and does not cause too much `line-height` change next to a text.

Reply:
so is it a ratio of some numbers or is it a random number?

Reply:
**I will revert this change and change the padding of button slightly instead.**

Not random: meant that a round circled icon is 1.1284 x larger than a square of a same area:

<img width="296" alt="Screenshot 2023-06-29 at 13 11 42" src="https://github.com/contember/interface/assets/387611/f3e67cdb-2ec4-41e4-bae7-27145ba6d9a7">

**Issue I wanted to handle: make buttons nice when round with and without the label:**

![Screenshot 2023-06-29 at 13 00 27](https://github.com/contember/interface/assets/387611/05ef817a-5e41-4dcf-aec3-06b4abed26c5)

Because it was ugly:

![Screenshot 2023-06-29 at 13 00 54](https://github.com/contember/interface/assets/387611/0e01b591-f167-43b7-81cf-e7519152f722)

Reply:
Done